### DISCUSSION TITLE
Adding drkhrse_miyoo_bezels + Jeltr0n optimizations on default with a toggle button to enable/disable

### DISCUSSION THREAD

**Discussion by Amiga500** [OnionUI/Onion#1000] [Ideas]
why not adding drkhrse_miyoo_bezels + Jeltr0n optimizations on default with a toggle button to enable/disable?

**Comment by Aemiii91** [comment]
We have this planned:
- https://github.com/OnionUI/Onion/issues/766

**Reply by Amiga500** [reply]
Fantastic, thanks!

**Reply by mlopezmad** [reply]
> We have this planned:
> 
> - https://github.com/OnionUI/Onion/issues/766

I wanted to ask if anything had been done about this, since I just asked @schmurtzm

**Reply by schmurtzm** [reply]
Nothing has been done on this subject for now.  But I like this idea ;)
